Prepare release version

milafrerichs opened the pull request:
## What does this PR do?
Prepares for release as a standalone tool. 

Remove the debugger from the viewer and use the default geonode viewer.
Build the viewer.jsx which does hold the abstraction from the react components.
Next step is to release all dependencies as a package and upload to npm to be used with geonode.

milafrerichs committed:
Do not use Debugger; build viewer

remove the debugger from the viewer and use the default geonode viewer
build the viewer.jsx which does hold the abstraction from the react
components

milafrerichs committed:
Merge branch 'master' into viewer-build

bartvde commented on viewer.jsx:
```diff
@@ -1,19 +1,23 @@
-    this.mapConfig = config;
+    this.mapConfig = value;
+  }
+  set setProxy(value) {
```
should both setters have the same signature? i.e. if this is called setProxy, should the other one be called setMapConfig instead?

milafrerichs replied:
yes, you're right. I addressed that a little different in the next commit.

milafrerichs committed:
Align setter signature

be  more precise that the class variables are private and only setters
are available

milafrerichs commented:
what do you think about the new setters @bartvde

bartvde commented:
perfect thanks @milafrerichs

milafrerichs commented:
thanks for your review @bartvde
